be/src/exec/olap_scanner.cpp

```diff
@@ -245,8 +246,9 @@ Status OlapScanner::_init_return_columns() {
                 break;
             }
         }
-        if (has_replace_col) {
-            _return_columns.push_back(_tablet->tablet_schema().sequence_col_idx());
+        if (auto sequence_col_idx = _tablet->tablet_schema().sequence_col_idx();
+            has_replace_col && std::find(_return_columns.begin(), _return_columns.end(), sequence_col_idx) == _return_columns.end()) {
+            _return_columns.push_back(sequence_col_idx);
```

Comment:
What case may cause sequence_col_idx insert to return columns duplicately

Reply:
it's a bug, i fixed it.
if you 'set show_hidden_columns = true;', select * from unique_table with sequnece column. In Debug mode will core.